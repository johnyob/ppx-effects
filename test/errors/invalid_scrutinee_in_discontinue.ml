let () =
  match%discontinue () with
  | _ -> ()
  | [%effect? _] -> ()
;;