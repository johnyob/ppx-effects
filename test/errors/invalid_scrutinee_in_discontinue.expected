File "invalid_scrutinee_in_discontinue.ml", line 2, characters 20-22:
Error: ppx_effects: invalid match%discontinue scrutinee. Expected a `continuation exception` expression.
